shardingsphere-underlying/shardingsphere-common/src/main/java/org/apache/shardingsphere/underlying/common/util/StringUtil.java

```diff
@@ -52,4 +52,14 @@ public static boolean isIntValue(final String value) {
             return false;
         }
     }
+
+    /**
+     * Judge is Null value or empty.
+     *
+     * @param value to be judged string value
+     * @return is long value or not
+     */
+    public static boolean isNullOrEmpty(final Object value) {
+        return value == null || "".equals(value.toString());
+    }
```

Comment:
Guava already this method, please reuse that one

Reply:
And according to style guide, null should be before ==

Reply:
> And according to style guide, null should be before ==

Sorry, I refer to the Objects. isNull() method.